cmake/toolchains/c.haiku.common.cmake

```diff
@@ -0,0 +1,9 @@
+set(CMAKE_SYSTEM_NAME Haiku)
+
+if(NOT CMAKE_SYSTEM_VERSION)
+    message(FATAL_ERROR "Need to set CMAKE_SYSTEM_VERSION to target version (uname -r) on cmd line")
+endif()
+
+find_program(CMAKE_C_COMPILER NAMES ${COMPILER_PREFIX}-gcc)
```

Comment:
Where is `COMPILER_PREFIX` set?

Reply:
this whole file is not required and can be removed. You are not crosscompiling and hence don't need a toolchain file.